tools/gen_struct_info.py

```diff
@@ -396,7 +396,7 @@ def inspect_code(headers, cpp_opts, structs, defines):
   # -Oz optimizes enough to avoid warnings on code size/num locals
   cmd = [shared.EMCC] + cpp_opts + ['-o', js_file[1], src_file[1],
                                     '-O0', '--js-opts', '0', '--memory-init-file', '0',
-                                    '-Werror', '-Wno-format',
+                                    '-Werror', '-Wno-format', '-Wno-error=version-check',
```

Comment:
Perhaps put this below along with `-Wno-deprecated` and `-Wno-fastcomp` as it is kind serving the same purpose?

Won't you have to do the same in tools/system_libs.py where the libraries are built?

Reply:
Moved it below.

Also, I didn't notice any other errors during my testing so dunno.